zebra/zebra_nhg.c

```diff
@@ -3553,6 +3557,15 @@ void zebra_nhg_dplane_result(struct zebra_dplane_ctx *ctx)
 				"Failed to uninstall Nexthop ID (%u) from the kernel",
 				id);
 
+		nhe = zebra_nhg_lookup_id(id);
```

Comment:
what's the relationship between this new code and the specific problem of adding another refcount to nhgs?